README.md

```diff
@@ -45,8 +45,10 @@ followed by
 4. Run your project (`Cmd+R`)<
 
 ## Android setup
+[Autolinking][autolinking]  unfortunately isn't supported for Android, because the SDK makes use of [TurboModules][turbomodules] to speed up app cold starts. 
```

Comment:
This is the 🥩 of the PR

Reply:
Do we want to link to the issue discussion on RN?